renovate.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ]
```

https://docs.renovatebot.com/modules/manager/git-submodules/

```suggestion
  ],
  "git-submodules": {
    "enabled": true
  }
```